Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -554,24 +563,17 @@ def issue_upload_command(issue_id, upload, attachment_name=None):
 
 
 def upload_file(entry_id, issue_id, attachment_name=None):
-    headers = {
-        'X-Atlassian-Token': 'no-check'
-    }
     file_name, file_bytes = get_file(entry_id)
-    res = requests.post(
-        url=BASE_URL + f'rest/api/latest/issue/{issue_id}/attachments',
-        headers=headers,
+    return jira_req(
+        method='POST',
+        resource_url=f'rest/api/latest/issue/{issue_id}/attachments',
+        headers={
+            'X-Atlassian-Token': 'no-check'
+        },
         files={'file': (attachment_name or file_name, file_bytes)},
-        auth=(USERNAME, API_TOKEN or PASSWORD),
-        verify=USE_SSL
+        resp_type='json'
     )
 
-    if not res.ok:
-        return_error(f'Failed to execute request, status code:{res.status_code}\nBody: {res.text}'
-                     + "\nMake sure file name doesn't contain any special characters" if res.status_code == 500 else "")
```

Comment:
You don't want to add this error for 500 cases?

Reply:
good catch, added it